README.md

```diff
@@ -4,7 +4,16 @@ CakePHP 3 Codeception Module
 [![Build Status](https://api.travis-ci.org/cakephp/codeception.png)](https://travis-ci.org/cakephp/codeception)
 [![License](https://poser.pugx.org/cakephp/codeception/license.svg)](https://packagist.org/packages/cakephp/codeception)
 
-A [codeception](http://codeception.com) module to test your CakePHP 3 powered application.
+***What?*** A [codeception](http://codeception.com) module to test your CakePHP 3 powered application.
+
+***Why?*** Using Codeception with CakePHP opens up a whole new set of testing capabilities for your app letting you do
+
+- **front-end testing** _(i.e. browser-based workflow tests)_
+  - [x] **[[Functional Tests]](http://codeception.com/docs/05-FunctionalTests)**  - does our Cake App run the way we expect it to
+  - [x] **[[Acceptance Tests]](http://codeception.com/docs/04-AcceptanceTests)**  - does our Cake App do what the customer/user expects it to
+- **back-end testing** _(i.e. direct app function calls)_
+   - [x] **[[API Tests]](http://codeception.com/docs/10-WebServices)** - does my Cake App's API work 
+   - [ ] **[[Unit Tests]](http://codeception.com/docs/06-UnitTests)** - do our App's internal functions do what they should
```

Comment:
While I like how it gives more details right off the bat, I personally don't really like the 'marketing' tone it has.

I suggest instead something like:

```
A [codeception](http://codeception.com) module to test your CakePHP 3 
powered application. Codeception opens up a whole new set of testing 
capabilities:
```

And for the list, I wouldn't split (front/back), would drop the checkboxes and styles. I would also follow this template:

```
- [Acceptance Tests]() - real browser tests to cover standard but complex 
scenarios from a user's perspective.
```